docker-compose/run_selenium_tests_local_keybase.sh

```diff
@@ -0,0 +1,214 @@
+
+BB20_TEAM_VAULT_PASSFILE="/cygdrive/k/team/bb20/infrastructure/creds/vault_pw.txt"
+BB20_DEPLOYMENT_VAULTFILE="../bluebutton-web-deployment/vault/env/test/vault.yml"
+BB20_ENABLE_REMOTE_DEBUG=True
+
+DJANGO_FHIR_CERTSTORE="/certstore"
+CERTSTORE_TEMPORARY_MOUNT_PATH="/tmp/certstore"
+CERT_FILENAME="client_data_server_bluebutton_local_integration_tests_certificate.pem"
+KEY_FILENAME="client_data_server_bluebutton_local_integration_tests_private_key.pem"
+
+# Backend FHIR server to use for selenium tests with FHIR requests:
+FHIR_URL="https://prod-sbx.bfd.cms.gov"
+
+# List of tests to run. To be passed in to runtests.py.
+TESTS_LIST="apps.integration_tests.selenium_tests.SeleniumTests"
+
+
+# Echo function that includes script name on each line for console log readability
+echo_msg () {
+  echo "$(basename $0): $*"
+}
+
+# main
+echo_msg
+echo_msg RUNNING SCRIPT:  ${0}
+echo_msg
+
+DJANGO_USER_ID_SALT=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_django_user_id_salt" | awk '{print $2}')
+DJANGO_USER_ID_ITERATIONS=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_django_user_id_iterations" | awk '{print $2}')
+
+DJANGO_PASSWORD_HASH_ITERATIONS=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_django_password_hash_iterations" | awk '{print $2}')
+DJANGO_SLSX_CLIENT_ID=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_slsx_client_id" | awk '{print $2}')
+DJANGO_SLSX_CLIENT_SECRET=$(ansible-vault view --vault-password-file=${BB20_TEAM_VAULT_PASSFILE} ${BB20_DEPLOYMENT_VAULTFILE} | grep "^vault_env_slsx_client_secret" | awk '{print $2}')
+
+# Set bash builtins for safety
+set -e -u -o pipefail
+
+USE_MSLSX=true
+DOCKER_COMPOSE_SERVICE="tests"
+
+# Parse command line option
+if [ $# -eq 0 ]
+then
+  echo "Use MSLSX for identity service."
+else
+  if [[ $1 != "slsx" && $1 != "mslsx" ]]
+  then
+    echo
+    echo "COMMAND USAGE HELP"
+    echo "------------------"
+    echo
+    echo "  Use one of the following command line options for the type of test to run:"
+    echo
+    echo "    slsx  = use SLSX for identity service."
+    echo
+    echo "    mslsx (default) = use MSLSX for identity service."
+    echo
+    exit 1
+  else
+    if [ $1 == "slsx" ]
+    then
+      USE_MSLSX=false
+      DOCKER_COMPOSE_SERVICE="tests_w_slsx"
+      HOSTNAME_URL="http://bb2slsx:8000"
+    fi
+  fi
+fi
+
+echo "DOCKER_COMPOSE_SERVICE: " ${DOCKER_COMPOSE_SERVICE}
+echo "HOSTNAME_URL: " ${HOSTNAME_URL}
+
+# Set KeyBase ENV path based on your type of system
+SYSTEM=$(uname -s)
+
+echo_msg " - Setting Keybase location based on SYSTEM type: ${SYSTEM}"
+echo_msg
+
+if [[ ${SYSTEM} == "Linux" || ${SYSTEM} == "Darwin" ]]
+then
+  keybase_env_path="/keybase"
```

Comment:
For mac could we make this: `/Volumes/keybase` instead?

Reply:
Don't have a Mac here, if that's the default keybase mount, we change to that....